micro-ui/web/micro-ui-internals/packages/modules/core/src/components/PrivacyComponent.js

```
@@ -34,7 +34,7 @@ const PrivacyComponent = ({ onSelect, formData, control, formState, ...props })
   return (
     <React.Fragment>
       <div className="digit-privacy-checkbox">
-        <CheckBox label={t("ES_BY_CLICKING")} checked={isChecked} onChange={handleCheckboxChange}></CheckBox>
+        <CheckBox label={t("ES_BY_CLICKING")} checked={isChecked} onChange={handleCheckboxChange} id={"privacy-component-check"}></CheckBox>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)